CMakeLists.txt

```diff
@@ -14,8 +15,17 @@ SET(ENABLE_AVX2 "ON" CACHE STRING "ENABLE_AVX2")
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c11")
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
 
+set(CMAKE_MACOSX_RPATH 1)
+
 set_property(SOURCE BlockedBloomFilter.cpp APPEND_STRING PROPERTY COMPILE_FLAGS " -funroll-loops")
 
+if(APPLE AND CMAKE_SYSTEM_PROCESSOR MATCHES "arm")
+       message("Disabling AVX2 instructions on arm64")
+       set(ENABLE_AVX2 "OFF")
```

Comment:
This change actually broke the compilation on OSX ARM64:
```
ERR) arm64-apple-darwin20.0: arm64-apple-darwin20.0: error: unsupported option '-mno-avx2' for target 'arm64-apple-darwin20.0.0'[0m
22:39:20 [32mBIOCONDA INFO[0m (ERR) error: unsupported option '-mno-avx2' for target 'arm64-apple-darwin20.0.0
```

More info at https://github.com/bioconda/bioconda-recipes/pull/52846